### DISCUSSION TITLE
Local network not working for AirPlay

### DISCUSSION THREAD

**Discussion by Jiggynut** [PoomSmart/YTABConfig#114] [Q&A]
![IMG_0653](https://github.com/PoomSmart/YTABConfig/assets/123044260/08d3a257-69a0-4855-8b58-96aa94d9097a)

**Comment by timfea926** [comment]
I can confirm I have the same problem and have started several threads on other YT repositories related to this very problem: https://github.com/qnblackcat/uYouPlus/issues/1395, https://github.com/arichornlover/uYouEnhanced/issues/514
